docs/guides/postgres/quickstart/quickstart.md

```diff
@@ -37,35 +38,70 @@ NAME    STATUS  AGE
 demo    Active  5s
 ```
 
+#### Install pgAdmin:
```

Comment:
Same here, Just use a `List` instead of dividing into a different section of `Before you Begin`.

Reply:
Moved out of Before You Begin